frontend/pages/Article.tsx

```diff
@@ -69,30 +69,32 @@ const articleContainer = css`
 
 const Article: React.SFC<{
     data: ArticleProps;
-}> = ({ data }) => (
-    <div>
-        <Header nav={data.NAV} pillar={data.CAPI.pillar} />
-        <main className={articleWrapper}>
-            <Container className={articleContainer}>
-                <article>
-                    <ArticleBody CAPI={data.CAPI} config={data.config} />
-                    <div className={secondaryColumn} />
-                </article>
-                <MostViewed sectionName={data.CAPI.sectionName} />
-            </Container>
-        </main>
+}> = ({ data }) => {
+    return (
```

Comment:
I'm probably missing something, but why wrap this in return here?

Reply:
Probably shouldn't be there.